app/main.py

```diff
@@ -1,8 +1,29 @@
+from __future__ import annotations
+
+
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
+
+    @classmethod
+    def find_by_name(cls, name: str) -> Person | None:
+        return cls.people.get(name)
```

Comment:
There is no need for this classmethod, you should get value from `people` attribute from the `Person` class directly.